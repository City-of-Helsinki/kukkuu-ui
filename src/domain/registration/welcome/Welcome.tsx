import React, { FunctionComponent } from 'react';
import { useTranslation } from 'react-i18next';

import tadaImage from '../../../assets/icons/svg/tada.svg';
import Button from '../../../common/components/button/Button';
import Icon from '../../../common/components/icon/Icon';
import styles from './welcome.module.scss';
import homeFormStyles from '../../home/form/homePreliminaryForm.module.scss';

const Welcome: FunctionComponent = () => {
  const { t } = useTranslation();
  return (
    <div className={styles.welcome}>
      <h1>{t('registration.welcome.hero.header')}</h1>
      <Icon src={tadaImage} className={styles.tada} alt="Tada!" />
      <form>
        <Button className={homeFormStyles.submitButton} type="submit">
          {t('common.profile.goToProfile.buttonText')}
        </Button>
      </form>
    </div>
  );
};

export default Welcome;
