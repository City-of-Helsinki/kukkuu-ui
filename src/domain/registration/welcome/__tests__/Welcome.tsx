import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Welcome from '../Welcome';

it('renders snapshot correctly', () => {
  const element = shallow(<Welcome />);
  expect(toJson(element)).toMatchSnapshot();
});
