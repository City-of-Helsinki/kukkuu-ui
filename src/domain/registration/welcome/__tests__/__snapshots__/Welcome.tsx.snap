// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot correctly 1`] = `
<div
  className="welcome"
>
  <div
    className="hero"
  >
    <div
      className="heroContainer"
    >
      <h1>
        registration.welcome.hero.header
      </h1>
      <Icon
        alt="Tada!"
        className="tada"
        src="tada.svg"
      />
      <form>
        <Button
          className="submitButton"
          type="submit"
        >
          common.profile.goToProfile.buttonText
        </Button>
      </form>
    </div>
  </div>
</div>
`;
